production/helm/loki/templates/statefulset.yaml

```diff
@@ -41,6 +41,10 @@ spec:
     {{- end }}
       securityContext:
         {{- toYaml .Values.securityContext | nindent 8 }}
+      initContainers:
+        {{- if .Values.initContainers }}
```

Comment:
This if check doesn't provide any value here. If `.Values.initContainers` is `null` or empty string then this will render `initContainers` as an invalid non-array type. Either put the if around the entire `initContainers` block or remove it.